frontend/src/App.vue

```diff
@@ -62,11 +62,12 @@ export default {
   methods: {
     ...mapMutations('auth', ['setLoading']),
     ...mapActions('auth', ['getJwtToken', 'getUserInfo', 'logout']),
+    ...mapActions('app', ['getLookupInfo']),
   },
   async created() {
     this.setLoading(true);
     this.getJwtToken().then(() =>
-      Promise.all([this.getUserInfo()])
+      Promise.all([this.getUserInfo(), this.getLookupInfo()])
```

Comment:
Consider refactoring to async/await syntax, which is preferred.

There's no need to wait for resolutions of getUserInfo and getLookupInfo since they appear to be called in "fire and forget" fashion.

This function's promise chain is redundant.

The following is the equivalent of this function:
```
async created() {
    this.setLoading(true);

    try {
      await this.getJwtToken();
    } catch (e) {
      if(! e.response || e.response.status !== HttpStatus.UNAUTHORIZED) {
        this.logout();
        this.$router.replace({name: 'error', query: { message: `500_${e.data || 'ServerError'}` } });
      }
    }

    this.getUserInfo();
    this.getLookupInfo();
    
    this.setLoading(false);
  }
```